radicle-surf/src/commit.rs

```diff
@@ -91,15 +120,13 @@ impl<'repo> TryFrom<git2::Signature<'repo>> for Author {
     fn try_from(signature: git2::Signature) -> Result<Self, Self::Error> {
         let name = str::from_utf8(signature.name_bytes())?.into();
         let email = str::from_utf8(signature.email_bytes())?.into();
-        let time = signature.when();
+        let time = signature.when().into();
 
         Ok(Author { name, email, time })
     }
 }
 
-/// `Commit` is the static information of a [`git2::Commit`]. To get back the
-/// original `Commit` in the repository we can use the [`Oid`] to retrieve
-/// it.
+/// `Commit` is the metadata of a [Git commit](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects).
```

Comment:
I prefer to use the footers for links to make it more readable :+1: 
```suggestion
/// `Commit` is the metadata of a [Git commit][git-commit]
///
/// [git-commit]: https://git-scm.com/book/en/v2/Git-Internals-Git-Objects
```